clouduniqueid/clouds/aws/__init__.py

```diff
@@ -67,39 +86,51 @@ def get_unique_id(
                 logger.error(f"AWS{errorMsg} keys required in data parameter")
                 raise ValueError(
                     f"Invalid parameters provided, AWS{errorMsg} keys required in data parameter,\
-                         uniqueId format for resource {service} {resourceType} is {uniqueIdFormat}",
+                        uniqueId format for resource {service} {resourceType} is {uniqueIdFormat}",
                 )
 
             else:
-                if uniqueIds[service][resourceType].startswith('hashlib'):
-                    if resourceType == 'acl':
+                if uniqueIds[service][resourceType].startswith("hashlib"):
+                    if resourceType == "acl":
                         try:
                             data = "{}:{}:{}:{}:{}:{}:{}:{}".format(
                                 accountId,
-                                data['owner'],
-                                data['ownerid'],
```

Comment:
<b>Vulnerability:</b> Detected the use of eval(). eval() can be dangerous if used to evaluate dynamic content. If this content can be input from outside the program, this may be a code injection vulnerability. Ensure evaluated content is not definable by external sources.<br/>
                            <b>Lines</b>: <b>Lines</b>: 113 to 113<br/>
                            <b>OWASP:</b> A03:2021 - Injection<br/>
                            <b><a href="https://console.cloudanix.com/code?t=vulnerabilities" target='_blank'>Click here for more details</a></b>